---
title: "Software and data packages"
date: "2019-01-02"
---

## Bioconductor 

#### Software 

- [R/qsmooth](http://bioconductor.org/packages/qsmooth): R package available that implements a generalization of quantile normalization, referred to as smooth quantile normalization (**qsmooth**), which is based on the assumption that the statistical distribution of each sample should be the same (or have the same distributional shape) within biological groups or conditions. ([Hicks et al., 2018. _Biostatistics_](https://academic.oup.com/biostatistics/article-abstract/19/2/185/3949169))
- [R/quantro](http://www.bioconductor.org/packages/release/bioc/html/quantro.html): R package available on Bioconductor to test for global differences between groups of distributions to decide when to use quantile normalization. ([Hicks and Irizarry, 2015. _Genome Biology_](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-015-0679-0))

#### Data packages

- [bodymapRat](http://bioconductor.org/packages/bodymapRat): R data package that contains an SummarizedExperiment from the [Yu et al. (2013)](https://www.ncbi.nlm.nih.gov/pubmed/24510058) paper that performed the rat BodyMap across 11 organs and 4 developmental stage. 
- [TENxPBMCData](http://bioconductor.org/packages/TENxPBMCData): R data package that contains a set of SingleCellExperiment objects with single-cell RNA-sequencing data from peripheral blood mononuclear cells generated by 10X Genomics. 

## GitHub

#### Software 

- [R/mbkmeans](https://github.com/drisso/mbkmeans): R/Bioconductor package implementing the mini-batch optimization for _k_-means clustering proposed in [Sculley (2010)](https://www.eecs.tufts.edu/~dsculley/papers/fastkmeans.pdf) for large datasets. The mini batch _k_-means algorithm can be run with data stored in memory or on disk (e.g. HDF5 file format).
- [R/methylCC](https://github.com/stephaniehicks/methylCC): R package available to estimate the cell composition of whole blood in DNA methylation samples in microarray or sequencing platforms
- [R/quantroSim](https://github.com/stephaniehicks/quantroSim): Supporting data simulation R-package for the *quantro* R-package to simulate gene expression and DNA methylation data.
- [R/explainr](https://github.com/hilaryparker/explainr): translates S3 objects into text using standard templates in a simple and convenient way. 
- [postMUT](https://github.com/stephaniehicks/postMUT): A tool implemented in Perl and R to predict the functionality of missense mutations.


#### Data packages

- [trapnell2014myoblasthuman](https://github.com/stephaniehicks/trapnell2014myoblasthuman): R data package that contains an ExpressionSet object from Trapnell et al. (2014) that performed a time-series experiment of bulk and single cell RNA-Seq at four time points in differentiated primary human myoblasts. 
- [patel2014gliohuman](https://github.com/willtownes/patel2014gliohuman): R data package that contains a SummarizedExpression object from Patel et al. (2014) with single cell and bulk RNA-Seq data on five human glioblastoma tumors. 
- [colonCancerWGBS](https://github.com/genomicsclass/colonCancerWGBS): Cov files produced from [Bismark](http://www.bioinformatics.babraham.ac.uk/projects/bismark/) after mapping six paired tumor-normal WGBS samples from Ziller et al. (2013) PMID: 23925113. Only chr22. 
- [myAffyData](https://github.com/stephaniehicks/mycAffyData): AffyBatch object from an experiment using P493-6 cells expressing low or high levels of c-Myc. Data from Loven et al. (2012) Cell 151: 476-482.
- [BackgroundExperimentYeast](https://github.com/stephaniehicks/BackgroundExperimentYeast): AffyBatch object from an experiment to measure NSB and optical noise in yeast.

